source/core_http_client.c

```diff
@@ -43,19 +43,6 @@
  */
 static uint32_t getZeroTimestampMs( void );
 
-
-HTTPStatus_t HTTPClient_SendHttpData( const TransportInterface_t * pTransport,
-                                      HTTPClient_GetCurrentTimeFunc_t getTimestampMs,
-                                      const uint8_t * pData,
-                                      size_t dataLen );
-
-
-HTTPStatus_t HTTPClient_SendHttpHeaders( const TransportInterface_t * pTransport,
-                                         HTTPClient_GetCurrentTimeFunc_t getTimestampMs,
-                                         HTTPRequestHeaders_t * pRequestHeaders,
-                                         size_t reqBodyLen,
-                                         uint32_t sendFlags );
-
```

Comment:
Honestly surprised that the double declaration of this didn't throw a warning or an error?

Reply:
I do not think declarations throw errors - you can declare a function as many time as you want. Definitions on the other hand would be a big no no :)